tests/test_cmd_run.py

```diff
@@ -35,8 +35,13 @@ def test_generate(self):
         database = data.Base(self.dbdir)
         target = data.Target(arches={'x86_64'}, trees={'rhel7'}, sources=None)
         baserun = run.Base(database, target, None)
+        variables = {
+            n: i['default']
+            for n, i in database.variables.items()
```

Comment:
What do `n` and `i` stand for? I'm used to either having clear names, or using `k` and `v` (for key and value) when it's small and it doesn't need full names.

Reply:
Yeah, `k` and `v` would probably be better. They stand for "name" and "info". I got entangled in some other idea and did it this way in kpet.cmd_run, and then did it the same way here. Let's leave it like this for now, and it will be overwritten by the work needed for FASTMOVING-1377.